sktime/transformations/series/summarize.py

```diff
@@ -45,45 +45,50 @@ class WindowSummarizer(BaseTransformer):
         symbols:
 
         ``z`` = time stamp that the window is summarized *to*.
+
         Part of the window if `lag` is between 0 and `1-window_length`, otherwise
         not part of the window.
+
         ``x`` = (other) time stamps in the window which is summarized
+
         ``*`` = observations, past or future, not part of the window
 
         The summarization function is applied to the window consisting of x and
         potentially z.
 
         For `window = [1, 3]`, we have a `lag` of 1 and
         `window_length` of 3 to target the three last days (exclusive z) that were
-        observed. Summarization is done across windows like this:
-        |-------------------------- |
+        observed. Summarization is done across windows like this::
+
+        |---------------------------|
         | * * * * * * * * x x x z * |
         |---------------------------|
 
         For `window = [0, 3]`, we have a `lag` of 0 and
         `window_length` of 3 to target the three last days (inclusive z) that
-        were observed. Summarization is done across windows like this:
-        |-------------------------- |
+        were observed. Summarization is done across windows like this::
+
+        |---------------------------|
         | * * * * * * * * x x z * * |
         |---------------------------|
 
 
-        Special case ´lag´: Since lags are frequently used and window length is
+        Special case ``lag``: Since lags are frequently used and window length is
         redundant, you only need to provide a list of `lag` values.
-        So `window = [1]` will result in the first lag:
+        So `window = [1]` will result in the first lag::
 
-        |-------------------------- |
+        |---------------------------|
         | * * * * * * * * * * x z * |
         |---------------------------|
 
-        And `window = [1, 4]` will result in the first and fourth lag:
+        And `window = [1, 4]` will result in the first and fourth lag::
 
-        |-------------------------- |
+        |---------------------------|
         | * * * * * * * x * * x z * |
         |---------------------------|
 
-        key: either custom function call (to be
-                provided by user) or str corresponding to native pandas window function:
+        key: either custom function call (to be provided by user) or
+            str corresponding to native pandas window function:
```

Comment:
The comment is actually against line 109, but Github doesn't allow me to comment on unchanged line. SHouldn't `truncate` be one level higher than `key` or `second_value`?

Reply:
Yup, I agree. The previous author may have missed the indentation correction. anything else you think `window_summarizer`'s docstring should be edited?

Reply:
No, nothing else that I can think of right now.